draft-ietf-rtcweb-jsep.xml

```diff
@@ -2486,15 +2497,17 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                   description, this MUST result in an error.</t>
                 <t>For each specified RTP header extension, establish a mapping between
                   the extension ID and URI. If any indicated RTP header extension is unknown, this
-                  MUST result in an error. [TODO: ref]</t>
+                  MUST result in an error. See section 6 of <xref target="RFC5285"/>.</t>
```

Comment:
Elsewhere in the document, I have used the 'as described in X' style, as opposed to the 'See X' style here. Would prefer to be consistent throughout, but don't feel super strongly.